db/schema.rb

```diff
@@ -325,6 +326,11 @@
     t.boolean "vehicle_income"
   end
 
+  create_table "snap_supplements", force: :cascade do |t|
```

Comment:
Is this leftover from the sketch I was doing? not finding anywhere where we're using it, may just be leftover in a local DB